CodeEdit.xcodeproj/project.pbxproj

```diff
@@ -3730,7 +3730,7 @@
 			repositoryURL = "https://github.com/CodeEditApp/CodeEditTextView.git";
 			requirement = {
 				kind = exactVersion;
-				version = 0.3.2;
+				version = 0.3.3;
```

Comment:
This version number is increasing much quicker than the main product version number. I just want to point this out.